assets/blocks/course-list-block/course-list.scss

```diff
@@ -49,3 +49,49 @@ $block: '.wp-block-sensei-lms-course-list';
 		padding: 1.25em;
 	}
 }
+
+.featured-badge {
```

Comment:
We should prefix these classes with `sensei-lms` in order to avoid collisions with themes or other plugins.

Reply:
Fixed here 83e1c4877a60feecb22d2d4f2fb5d588794c08a3